pkgs/applications/networking/seahub/default.nix

```
@@ -59,14 +55,21 @@ python.pkgs.buildPythonApplication rec {
     chardet
     pyjwt
     pycryptodome
+    pyopenssl
+    python-ldap
     qrcode
     pysearpc
-    seaserv
     gunicorn
     markdown
     bleach
+
+    seaserv
```

```suggestion
    (seaserv.override { inherit python3; })
```
something like this should achieve the same as above